susemanager-utils/susemanager-sls/src/tests/test_beacon_pkgset.py

```diff
@@ -12,10 +12,7 @@ def test_virtual():
     '''
     Test virtual function.
     '''
-    with patch('os.path.exists', MagicMock(return_value=False)):
-        assert pkgset.__virtual__() != pkgset.__virtualname__
-    with patch('os.path.exists', MagicMock(return_value=True)):
-        assert pkgset.__virtual__() == pkgset.__virtualname__
+    assert pkgset.__virtual__() == pkgset.__virtualname__
```

Comment:
I think this test is meaningless, I would remove it.

Reply:
Oh yes, now it's

Reply:
Removed with https://github.com/uyuni-project/uyuni/pull/4427/commits/9fe93cb39738efc3d559b031d24622b32eabfe20